### ISSUE TITLE
Image Auto Release Cron Job is failing

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#1563]
The cron job to regularly release our Kubeflow images is failing.
It runs in the kubeflow-releasing cluster;

Here is the error:

```
INFO|2018-09-19T00:05:44|/src/kubeflow/testing/py/kubeflow/testing/util.py|62| level=info msg="Creating environment \"image-release-centraldashboard-release-d8973b1-f6c7-427a\" with namespace \"default\", pointing to cluster at address \"https://35.226.49.107\""
INFO|2018-09-19T00:05:44|/src/kubeflow/testing/py/kubeflow/testing/util.py|62| level=error msg="load configuration: Current app uses unsupported spec version '0.2.0' (this client only supports 0.1.0)"
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/src/kubeflow/testing/py/kubeflow/testing/run_e2e_workflow.py", line 349, in <module>
    final_result = main()
  File "/src/kubeflow/testing/py/kubeflow/testing/run_e2e_workflow.py", line 339, in main
    return run(args, file_handler)
  File "/src/kubeflow/testing/py/kubeflow/testing/run_e2e_workflow.py", line 189, in run
    util.run(["ks", "env", "add", env], cwd=w.app_dir)
  File "/src/kubeflow/testing/py/kubeflow/testing/util.py", line 74, in run
    " ".join(command), process.returncode), "\n".join(output))
subprocess.CalledProcessError: Command 'cmd: ks env add image-release-centraldashboard-release-d8973b1-f6c7-427a exited with code 1' returned non-zero exit status 1
```

I think this indicates that ksonnet is too old in the image.

**Comment by jlewi** [comment]
The pod used image:
gcr.io/kubeflow-ci/test-worker:latest
gcr.io/kubeflow-ci/test-worker@sha256:66fc77fc0f2cb1e388707da2e6f212c58e8778d21e02ea6194d563376c5a8fb2

**Comment by jlewi** [comment]
It looks like ks 0.11 is installed in that image

ks version
ksonnet version: 0.11.0
jsonnet version: v0.10.0
client-go version:

**Comment by jlewi** [comment]
So it looks like we have some ksonnet release apps that are using the older ksonnet spec (tf serving) and some (centra ui) are using the newer spec.

Simplest fix is to use K8s app spec 0.1.0 for all the workflows. 
It looks like each release workflow is in its own ksonnet APP we can probably consolidate them.

**Comment by jlewi** [comment]
Closing this issue. We are now using postsubmits and not the cron job to build images so that we can use test grid to track success & failure.
